app/assets/javascripts/bastion/components/views/bst-bookmark.html

```diff
@@ -0,0 +1,46 @@
+<ul>
+  <li ng-repeat="bookmark in bookmarks"><a ng-click="get(bookmark)" class="ng-scope">{{ bookmark.name }}</a></li>
+  <li class="divider"></li>
+  <li><a ng-click="add()" translate class="ng-scope">Bookmark this search</a></li>
+  <li>
+    <a ng-href="http://www.theforeman.org/manuals/1.10/index.html#4.1.5Searching" data-id="aid_manuals_1.10_index.html" rel="external" target="_blank"><i class="glyphicon glyphicon-question-sign icon-black"></i>
+      <span translate> Documentation</span>
+    </a>
+  </li>
+</ul>
```

Comment:
This should be using the classes from bootstrap's [dropdown](http://getbootstrap.com/components/#dropdowns) so it is styled correctly.  There is no padding currently because these classes are not being used:

![screen shot 2015-12-15 at 9 09 45 am](https://cloud.githubusercontent.com/assets/4116405/11812773/ce98d2a8-a30b-11e5-80a2-04ff5b646b81.png)